This bug in GCC 4.8.2 is fixed by
poky/meta/recipes-devtools/gcc/gcc-4.8/0048-PR58854_fix_arm_apcs_epilogue.patch

diff --git a/wrapper/compiler.h b/wrapper/compiler.h
index 0c01632..ec6c904 100644
--- a/wrapper/compiler.h
+++ b/wrapper/compiler.h
@@ -33,7 +33,7 @@
  * http://gcc.gnu.org/bugzilla/show_bug.cgi?id=58854
  */
 # ifdef __ARMEL__
-#  if GCC_VERSION >= 40800 && GCC_VERSION <= 40802
+#  if GCC_VERSION >= 40800 && GCC_VERSION <= 40801
 #   error Your gcc version produces clobbered frame accesses
 #  endif
 # endif
